### ISSUE TITLE
crash on holding value items

### ISSUE THREAD

**Issue by JayRPlayz** [homchom/recode#67]
sometimes, when i hold a value item, the description is broken, sometimes it crashes the game, when editing a value i crash instantly.
i sadly dont have the original modpack with the crashlogs, but think its something to do with "roughly enough items" mod.

**Comment by MrSam7K** [comment]
Try downloading latest build or just wait for official release, it's just around the corner!

**Comment by homchom** [comment]
Yeah I was gonna ask, does this happen when you download and use the latest build? Because this should be fixed

**Comment by JayRPlayz** [comment]
it is fixed, but theres a new issue
